account.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"math"
+	"time"
+
+	"github.com/cockroachdb/apd/v2"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	auth "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vesting "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+// Account is an internal representation of a genesis regen account
+type Account struct {
+	Address       sdk.AccAddress
+	TotalRegen    apd.Decimal
+	StartTime     time.Time
```

Comment:
Is start time supposed to be always set to mainnet for every account? That seems to make sense since I feel like their vesting begins always at `StartTime`. I worry there may be some failure mode if we have accounts with a `StartTime` in the future... Like maybe those users wouldn't be able to stake.

Reply:
`StartTime` should be in the future right? It isn't set to mainnet. I don't think that affects ability to delegate - we should check the vesting account logic to be sure

Reply:
Check the `BaseVestingAccount.TrackDelegation` logic. I think it should be fine.